### PR TITLE
CEDS-5343: Play 3.0 Upgrade

### PR DESCRIPTION
**Pull request by Dweik97**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -12,10 +12,7 @@ lazy val microservice = (project in file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtDistributablesPlugin)
   .settings(commonSettings: _*)
   .settings(
-    Test / unmanagedSourceDirectories := Seq(
-      (Test / baseDirectory).value / "test/unit",
-      (Test / baseDirectory).value / "test/utils"
-    ),
+    Test / unmanagedSourceDirectories := Seq((Test / baseDirectory).value / "test/unit", (Test / baseDirectory).value / "test/utils"),
```
Comment (udinobi) [inline build.sbt:15]: Do not know if this change is due to some auto reformatting, but honestly it was more readable as it was before

Comment (Dweik97) [reply build.sbt:15]: It is auto formatting, and I agree.

File: build.sbt [hunk 2]
```diff
@@ -31,23 +28,22 @@ lazy val microservice = (project in file("."))
...
-  .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
+  .disablePlugins(JUnitXmlReportPlugin) // Required to prevent https://github.com/scalatest/scalatest/issues/1427
 
-lazy val commonSettings = Seq(
-  scalaVersion := "2.13.8",
-  scalacOptions ++= scalacFlags,
-  libraryDependencies ++= Dependencies.compile ++ Dependencies.test
-)
+lazy val commonSettings =
+  Seq(scalaVersion := "2.13.12", scalacOptions ++= scalacFlags, libraryDependencies ++= Dependencies.compile ++ Dependencies.test)
```
Comment (udinobi) [inline build.sbt:34]: Do not know if this change is due to some auto reformatting, but honestly it was more readable as it was before

**Review by udinobi — APPROVED** [review]

File: build.sbt (hunk 2 above)
Comment (Dweik97) [reply build.sbt:34]: The only thing I changed in this file is the scalaVersion number and the silencing of the "private val ..." in scalaFlags, the rest is scalafmt being wacky😞

File: build.sbt (hunk 1 above)
Comment (udinobi) [reply build.sbt:15]: It's weird btw to have auto-formatting on build.sbt :raised_eyebrow:

**Pull request merged**